srcs/expansion/expansion.c

```
@@ -0,0 +1,24 @@
+/*                                                                            */
+/* ************************************************************************** */
+
+/*
+$? -> 前に終了したステータスの中身を文字列として取り出す
+$<変数名> -> 変数名に該当する変数を展開し、中身を文字列として取り出す
+*展開 -> カレントディレクト下のファイル・フォルダ名に展開することが出来る。
+シングルクォーテーションで囲まれている場合、展開は行わない
+ダブルクォーテーションで囲まれている場合、$変数の展開は行う
+*/
+
+expand_sp_var()
```

cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If expand_sp_var is a macro then please configure it.

(unknownMacro)